octavia-cli/octavia_cli/entrypoint.py

```diff
@@ -8,41 +8,58 @@
 import click
 from airbyte_api_client.api import workspace_api
 
+from .check_context import check_api_health, check_is_initialized, check_workspace_exists
+from .init import commands as init_commands
 from .list import commands as list_commands
 
-AVAILABLE_COMMANDS: List[click.Command] = [list_commands._list]
+AVAILABLE_COMMANDS: List[click.Command] = [list_commands._list, init_commands.init]
 
 
 @click.group()
 @click.option("--airbyte-url", envvar="AIRBYTE_URL", default="http://localhost:8000", help="The URL of your Airbyte instance.")
+@click.option(
+    "--workspace-id",
+    envvar="AIRBYTE_WORKSPACE_ID",
+    default=None,
+    help="The id of the workspace on which you want octavia-cli to work. Defaults to the first one found on your Airbyte instance.",
```

Comment:
This help message will be displayed when the user runs something like `octavia --help`, right? Just wanted to make sure that it is made clear to the user that this option is how they can specify a specific workspace

Reply:
Yes you're right, its displayed in when running `octavia --help`: 
```
Usage: octavia [OPTIONS] COMMAND [ARGS]...

Options:
  --airbyte-url TEXT   The URL of your Airbyte instance.
  --workspace-id TEXT  The id of the workspace on which you want octavia-cli
                       to work. Defaults to the first one found on your
                       Airbyte instance.
  --help               Show this message and exit.
  ```